src/platform/backends/shared/base_snapshot.cpp

```diff
@@ -78,12 +78,14 @@ std::shared_ptr<const mp::Snapshot> find_parent(const QJsonObject& json, const m
 }
 } // namespace
 
-mp::BaseSnapshot::BaseSnapshot(const std::string& name, const std::string& comment, // NOLINT(modernize-pass-by-value)
+mp::BaseSnapshot::BaseSnapshot(const std::string& name, const std::string& comment,
```

Comment:
Is this an artifact from using CLion? The comment makes no difference with my formatter.

Reply:
As discussed, this was an intentional way to tell clang-tidy not to make that particular suggestion. It prevents that line from being flagged to those of us using clang-tidy. 

BTW, clang-tidy comes with CLion, but I have confirmed that comment is read by clang-tidy and not CLion (I wasn't sure when we spoke).

Additionally, it documents to the reader that I consciously decided not to [pass by value and move](https://clang.llvm.org/extra/clang-tidy/checks/modernize/pass-by-value.html) there (because I prefer the benefit of *small string optimization* here, as that small string (char array) would need to be copied again to move the string).

Reply:
Ok, I will rebase and add it back in

Reply:
My impression was that you did not have any objection when we spoke, right? If you do, or if you are just not sure, that is of course fine and we can all discuss it further. It is going to be reviewed again, but we can discuss early too :slightly_smiling_face: 

If, on the other hand, it is all the same to you, would you mind restoring the comment?

Reply:
Oh crossed comments. Thanks!